app/views/users/sessions.py

```diff
@@ -141,6 +144,65 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        for speaker in session.speakers:
+            data += (speaker.name + ',') if speaker.name else ','
+            data += (session.title + ',') if session.title else ','
+            data += (speaker.email + ',') if speaker.email else ','
+            data += (speaker.sessions + ',') if speaker.sessions else ','
+            data += (speaker.mobile + ',') if speaker.mobile else ','
+            data += (speaker.organisation + ',') if speaker.organisation else ','
```

Comment:
@niranjan94 
Nothing should be worse than this user's organisation :)
![nothing-worst-than-this-organisation](https://cloud.githubusercontent.com/assets/17861054/26146476/f11d324c-3b0d-11e7-8f6d-15ee4c279f7c.png)

And since it is working so I assume all other would pass on.